web/js/scripts.js

```diff
@@ -27,12 +27,16 @@ var grouphub = (function ($) {
     var searchUsersOrGroups = function () {
         var $this = $(this),
             $searchContainer = $this.closest('.search_container'),
-            $searchResults = $searchContainer.next('ul');
-
-        var url = $searchResults.data('url');
-        var type = $searchContainer.find("select[name=search-type]").val();
-        if (type === 'group') {
-            url = url.replace('users', 'groups');
+            $searchResults = $searchContainer.next('ul'),
```

Comment:
Personally not a fan of using comma's to declare variables within the same stated scope. I like to put `var` (or, in ES2015, `let` and `const`) for each variable separately to be as explicit as possible.
This is a personal preference, feel free to ignore completely 😉